birdhouse/components/proxy/docker-compose-extra.yml

```diff
@@ -12,12 +12,10 @@ services:
     image: ${PROXY_IMAGE}
     container_name: proxy
     ports:
-      - "80:80"
-      - "443:${PROXY_SECURE_PORT}"
```

Comment:
I think the 443 could be left and is simply ignored.
Makes sense to keep this edit depending on other discussions regarding how the redirects are handled and how variables get defined.

Reply:
> I think the 443 could be left and is simply ignored.

True, but better not to open up ports if it's not necessary

> Makes sense to keep this edit depending on other discussions

Sure that makes sense